iroha/src/config.rs

```diff
@@ -35,6 +35,8 @@ pub struct Configuration {
     pub logger_configuration: LoggerConfiguration,
     /// Configuration for initial setup
     pub init_configuration: InitConfiguration,
+    /// `GenesisBlock` path.
+    pub genesis_block_path: Option<String>,
```

Comment:
Should I use `Option<PathBuf>`? 
In my option, unnecessary mutable path...

Reply:
Yes, sure, you can use path buff